worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -1230,8 +1219,8 @@ class PushDataHandler(val workerSource: WorkerSource) extends BaseMessageHandler
     }
     val mountPoint = fileWriter.flusher.asInstanceOf[LocalFlusher].mountPoint
     val diskInfo = workerInfo.diskInfos.get(mountPoint)
-    val diskFull = diskInfo.status.equals(
-      DiskStatus.HIGH_DISK_USAGE) || diskInfo.actualUsableSpace < diskUsableSizes(mountPoint)
+    val diskFull =
+      diskInfo.status.equals(DiskStatus.HIGH_DISK_USAGE) || diskInfo.actualUsableSpace <= 0
```

Comment:
The failed cases are all due to a ut (CelebornHashCheckDiskSuite). After investigation, it‘s caused by the modification here.
Before this, actualUsableSpace is set by 
```val usableSpace = Math.max(workingDirUsableSpace - minimumReserveSize, 0)``` and always >=0.
This ut set reserve size to 0, so here check can pass. And I also tried changing the condition to ```diskInfo.actualUsableSpace < 0```, and ut passed.
I need to realize the goal of this ut and try to modify it later.